### ISSUE TITLE
TypeScript comments in type alias are displaced

### ISSUE THREAD

**Issue by calebmer** [prettier/prettier#1946]
```ts
function x({
  x,
  y,
}: {
  // Hello world.
  x: string,
  // Yoyo.
  y: string,
}) {}
```

=>

```ts
function x({
  x,
  y,
}: {
  x: string;
  y: string;
}) // Hello world.
// Yoyo.
{
}
```

https://prettier.github.io/prettier/#%7B%22content%22%3A%22function%20x(%7B%5Cn%20%20x%2C%5Cn%20%20y%2C%5Cn%7D%3A%20%7B%5Cn%20%20%2F%2F%20Hello%20world.%5Cn%20%20x%3A%20string%2C%5Cn%20%20%2F%2F%20Yoyo.%5Cn%20%20y%3A%20string%2C%5Cn%7D)%20%7B%7D%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22all%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22typescript%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%7D%7D

**Comment by vjeux** [comment]
https://github.com/prettier/prettier/issues/1694 is a similar issue.

**Comment by vjeux** [comment]
I just published 1.4.3 which fixes it. Thanks for reporting it!
